cmd/operator/main.go

```diff
@@ -61,8 +62,11 @@ func init() {
 
 func main() {
 	var debugLogging bool
+	var metricsAddr string
 
 	flag.BoolVar(&debugLogging, "debugLogging", false, "Log debug messages")
+	flag.StringVar(&metricsAddr, "metrics-bind-address", "0.0.0.0:8443",
+		"The address the metric endpoint binds to.")
```

Comment:
Please mention the proposed default (`8443`) in the argument description.

Reply:
nit, if you still want to add this